syntax/std_seq_array_helper.go

```diff
@@ -0,0 +1,206 @@
+package syntax
+
+import (
+	"github.com/arr-ai/arrai/rel"
+)
+
+// arrayContains check if array a contains b, and b can be rel.Value or rel.Array.
+func arrayContains(a rel.Array, b rel.Value) rel.Value {
```

Comment:
Use more meaningful parameter names here and below.

Reply:
Fixed.